services/task_store/service.go

```diff
@@ -459,15 +459,19 @@ func (ts *Service) handleTask(w http.ResponseWriter, r *http.Request) {
 		httpd.HttpError(w, fmt.Sprintf("invalid dot-view parameter %q", dotView), true, http.StatusBadRequest)
 		return
 	}
+
 	tmID := r.URL.Query().Get("replay-id")
 	if tmID == "" {
 		tmID = kapacitor.MainTaskMaster
 	}
+
 	tm := ts.TaskMasterLookup.Get(tmID)
+
 	if tm == nil {
 		httpd.HttpError(w, fmt.Sprintf("no running replay with ID: %s", tmID), true, http.StatusBadRequest)
 		return
 	}
+
```

Comment:
Can you revert these whitespace changes here? They don't seem relevant to the changes.

Reply:
Sure. Will fix all the whitespace and line breaks added.